sycl/doc/extensions/proposed/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -146,73 +143,89 @@ specified explicitly.
 
 [NOTE]
 ====
-If a `work_group_static` object is declared at function scope, the
+If a `work_group_static` object is declared at a block scope, the
 device local memory associated with the object will be identical for all
 usages of that function within the kernel. In cases where a function is called
 multiple times, developers must take care to avoid race conditions (e.g., by
 calling `group_barrier` before and after using the memory).
 ====
 
+Change to SYCL 2020 section `5.9.2 Common address space deduction rules`:
+Namespace scope: if the variable is `work_group_static` object,
+then the variable is assigned to the local address space.
+Otherwise normal rules applies.
```

Comment:
I like to avoid references to specific section numbers in the core spec when possible.  Can you instead move this to a new section in this extension document like:

> === Interaction with common address space deduction rules
>
> If a variable of type `work_group_static` is declared at namespace scope, the variable is assigned to the local address space.

BTW, why doesn't this apply also if the variable is declared at block scope?